ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js

```diff
@@ -80,6 +81,24 @@ const render = ({ txProps, contextProps } = {}) => {
       identities: {
         '0xAddress': {},
       },
+      internalAccounts: {
+        accounts: {
+          'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3': {
+            address: '0xAddress',
+            id: 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3',
+            metadata: {
+              name: 'Test Account',
+              keyring: {
+                type: 'HD Key Tree',
+              },
+            },
+            options: {},
+            methods: [...Object.values(EthMethod)],
+            type: EthAccountType.Eoa,
+          },
+        },
+        selectedAccount: 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3',
```

Comment:
This is quite a repeated mock we use in all tests, we should probably create this `internalAccounts` in a mock object inside the file `test/jest/mocks` and just import it wherever needed. Feel free to do it here or another PR

Reply:
I'll create a follow on PR to dynamically create the mocks as needed.